src/index.html

```diff
@@ -1,13 +1,318 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Museum landing page</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:wght@300;400;500;600;700&family=Montserrat:wght@400;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+
+  <body class="body">
+    <header class="header">
+      <div class="header__container">
+        <div class="header__top-actions top-actions">
+          <a
+            href="#menu"
+            class="menu-btn"
+            for="menutoggler"
+          >
+            <div class="menu"></div>
+            <div class="menu"></div>
+            <div class="menu"></div>
+          </a>
+          <a href="#" class="header__title">
+            NA
+            <br>
+            MU
+          </a>
+        </div>
+        <section class="museum" id="museum">
+          <div class="museum__container">
+            <h1 class="museum__title">
+              Худож
+              <span class="header__outline"> ній </span>
```

Comment:
```suggestion
              <span class="header__outline">ній </span>
```
Просто прибери тут пробіл

Reply:
соромно, що пробіл не побачив. все пофіксив, але завантажив, щоб подумати як пофіксити марджини широкі по бокам. бо в мене все на фіксованому тримається, і дуже не хочеться несучу конструкцію ламати, тому буду думати як краще пофіксити ширину сторінки від 320 до 640.